File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-3.js
prompt = require('prompt-sync')();

console.log(`
**** TABELA VERDADE ****
    1.  Operador AND
    2.  Operador OR
    3.  Operador NOT
**************************
`)

const opcao = Number(prompt('Escolha uma opção:'))
let num1
let num2


switch (opcao){
    case 1:
        num1 = Number(prompt('Digite o valor do numero 1:'))
        num2 = Number(prompt('Digite o valor do numero 2:'))
        console.log(num1 && num2)
        break

    case 2:
        num1 = Number(prompt('Digite o valor do numero 1:'))
        num2 = Number(prompt('Digite o valor do numero 2:'))
        console.log(num1 || num2)
        break
    case 3:
        num1 = Number(prompt('Digite o valor do numero:'))
        console.log(Number(!num1))
        break
    default:
        console.log('Opção Inválida')
}

File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_3.py
print(f'{4*'*'} TABELA VERDADE {4*'*'}\n'
      '1. Operador AND\n'
      '2. Operador OR\n'
      '3. Operador NOT\n'
      f'{27*'*'}')
opcao = int(input('Escolha uma opção:'))

match opcao:
    case 1:
        num_1 = bool(int(input('Digite o primeiro valor:')))
        num_2 = bool(int(input('Digite o segundo valor:')))

        print(f'Resultado: {num_1 and num_2}')
    
    case 2:
        num_1 = bool(int(input('Digite o primeiro valor:')))
        num_2 = bool(int(input('Digite o segundo valor:')))

        print(f'Resultado: {num_1 or num_2}')

    case 3:
        num_1 = bool(int(input('Digite o primeiro valor:')))

        print(f'Resultado: {not num_1}')